park_api/env.py

```diff
@@ -95,7 +95,7 @@ def determine_server_version():
     try:
         proc = subprocess.Popen(["git", "rev-list", "--all", "--count"], stdout=subprocess.PIPE)
         rev = proc.stdout.read().decode('utf-8').strip()
-        SERVER_VERSION = '0.1.{0}'.format(rev)
+        SERVER_VERSION = '0.2.{0}'.format(rev)
```

Comment:
Didn't we decide on incrementing this with breaking changes? The iOS version will not accept server responses with version != 0.1.x... I should probably change that.